src/Raven.Server/Documents/Sharding/ShardedDocumentDatabase.cs

```diff
@@ -139,8 +139,9 @@ public void HandleReshardingChanges()
                 }
 
                 // cleanup values
```

Comment:
Why `_confirmations` is a dictionary on long instead of the change vector directly?

Reply:
because it can be `process.MigrationIndex` (first `if` statement).